### DISCUSSION TITLE
multi horizon timeseries time_idx

### DISCUSSION THREAD

**Discussion by Pavan-Naga-207** [jdb78/pytorch-forecasting#1682] [Q&A]
In my dataset, i have multiple time series, like price of product in different states, districts and markets. and the length of time series are not constant. some time series are staring from 01-01-2011, and some are starting from 01-01-2015. so in this case how can i set the time_idx.

**Comment by benHeid** [comment]
This is more a question. Thus, I convert it to a discussion

**Comment by benHeid** [comment]
Did you tried to use groups for each of these time series? I.e., group 1 contains time series for district/state x, group 2 for district/state y, etc.?
